src/features/mcda/components/MCDALayerEditor/MCDALayerParameters/MCDALayerParameterRow/MCDALayerParameterRow.tsx

```diff
@@ -0,0 +1,31 @@
+import { TooltipTrigger } from '~components/TooltipTrigger';
+import { LAYERS_PANEL_FEATURE_ID } from '~features/layers_panel/constants';
+import s from './MCDALayerParameterRow.module.css';
+
+export type MCDALayerParameterRowProps = {
+  name: string;
+  tipText?: string;
+  children?: JSX.Element | JSX.Element[];
+};
+
+export function MCDALayerParameterRow({
+  name,
+  tipText,
+  children,
+}: MCDALayerParameterRowProps) {
+  return (
+    <div className={s.inputLine}>
+      <span className={s.inputLinelabel}>
+        {name}
+        {tipText && (
+          <TooltipTrigger
```

Comment:
`TooltipTrigger` should be wrapped with `Tooltip` component, please check existing app tooltips

Reply:
Tooltip should looks like this

```tsx
    <Tooltip placement="bottom">
      <TooltipTrigger>
        <div className={s.analyticsBadge}>
          {icon && icon}
          <span className={s.analyticsValue}>
            <Text type="caption">{value}</Text>
          </span>
        </div>
      </TooltipTrigger>
      <TooltipContent>{tooltip}</TooltipContent>
    </Tooltip>
```